com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerModelLoader.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using UnityEngine;
+using System.Threading.Tasks;
+
+#if MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT && KTX_PRESENT
+using Microsoft.MixedReality.OpenXR;
+using GLTFast;
+#endif // MROPENXR_PRESENT && (UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_ANDROID) && GLTFAST_PRESENT && KTX_PRESENT
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// A helper class which loads and caches controller models fetched from the controller's platform sdk
+    /// </summary>
+    public class ControllerModelLoader
```

Comment:
nit: should this itself be a `static` class? I think everything in it is `static` so far

Reply:
Done